web/templatetags/membership_tags.py

```
@@ -0,0 +1,35 @@
+User = get_user_model()
+register = template.Library()
+
+
+@register.filter
+def format_price(value: Optional[Union[float, int]] = None, period: Optional[str] = None) -> str:
+    """Format price based on billing period."""
+    if value is None:
+        return "$0"
+    if period == "yearly":
+        return f"${value}/year"
+    elif period == "monthly":
```

Ruff (0.8.2)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)